pdc.gemspec

```diff
@@ -3,6 +3,7 @@ lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'pdc/version'
 
+# afadsfdasf asdlfjdasljf lakjfldsaj l;sadjfl;asdjf lajsdfldsjal;fj asl;dkjfladsjf ;ljasf;ljsda;lfjdsal;sl;jslj;lj;lsjfla l;kdflksdj
```

Comment:
Metrics/LineLength: Line is too long. [132/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)